influxdb-meta/README-short.txt

```diff
@@ -0,0 +1 @@
+InfluxDB Enterprise is a highly available time series database.
```

Comment:
They want the little blurb to be <= 100 characters. Is there a way we can remove some of the words and get the same point across?